talentprotocol/web-dapp
# feat: Galxe google sheets

## Summary

This PR adds a class to upload wallet addresses of users with goals with a "doing" progress OR with 4 or more subscription requests to Google Sheets.

It also adds a job to run this every four hours.

## Notion link

https://www.notion.so/talentprotocol/Bounty-Google-Sheets-for-Galxe-932473200d764e318a8417b416f5392b

## How to test?

Run `GoogleDrive::Galxe::UploadWalletAddresses.new.call` and check Google Sheets:
- https://docs.google.com/spreadsheets/d/1gmkG1VMPrcSj9BfAzXuXJgzNRmo_AUM1SAb6uOwEC3k/edit#gid=0
- https://docs.google.com/spreadsheets/d/1RdxT-UYAfWlDeS7C7ygSC-MrZHzNzPttolWKhWo2rSs/edit#gid=0

**Delete data after running to clear sheet**
